sbag/generators/javascript/templates/custom_path/__path_name__.service.ts.jinja

```diff
@@ -0,0 +1,27 @@
+/// Generated from {{ self._TemplateReference__context.name | get_template_name_from_path }} by SBAG (https://github.com/vtanja/JSD)
+/// on {{ date }}
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+{% set path_lower = path_name | first_letter_lower %}
+{% set path_cap = path_name | capitalize %}
+{% set service_name = path_lower + 'Service' %}
+{% for entity in (new_controllers[path_cap] | get_custom_path_imports) %}
```

Comment:
Can you try to put here: 
`{% for entity in controller_imports[path_cap] %}` 
and see if it is different? That way we can remove extra functions.

Reply:
That way I get more imports than I actually need. I only need parameter entities, not return types as well.

Reply:
Sound's good